apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -703,3 +701,8 @@ def get_items_list(items_string: None | str) -> list[str]:
         return []
 
     return list(map(str.strip, items_string.split(",")))
+
+
+def full_unidecode(input_str: str) -> str:
+    """Does unidecode of input and replaces the unicode minus with the normal one."""
+    return unidecode(input_str.replace("\u2212", "-"))
```

Comment:
This does not feel like it belongs here - there is absolutely nothing sympy-specific about this function. It looks like you had it in `prairielearn.py` in a previous iteration; why the change?

Reply:
If put in prairielearn.py, it can't be imported here because of a circular import.

Reply:
Hmm, that's unfortunate. I'd rather use a workaround like doing `from prairielearn import full_unidecode` inside `evaluate_with_source`.

Reply:
In https://github.com/PrairieLearn/PrairieLearn/pull/8634/commits/66310efceeeac915c8199a5682f1631348e60d15 I was able to avoid this by importing the modules (I think).

Reply:
Looks like Black needs to reformat some things.